packages/eslint-plugin/src/rules/no-confusing-void-expression.ts

```diff
@@ -84,7 +84,7 @@ export default util.createRule<Options, MessageId>({
         const checker = parserServices.program.getTypeChecker();
         const tsNode = parserServices.esTreeNodeToTSNodeMap.get(node);
         const type = util.getConstrainedTypeAtLocation(checker, tsNode);
-        if (!tsutils.isTypeFlagSet(type, ts.TypeFlags.VoidLike)) {
+        if (!util.isTypeFlagSet(type, ts.TypeFlags.VoidLike)) {
```

Comment:
There's no real need to change all the existing `tsutils.isTypeFlagSet` locations to the new helper, is there?